src/test/java/org/assertj/core/presentation/CompositeRepresentation_Test.java

```diff
@@ -0,0 +1,62 @@
+package org.assertj.core.presentation;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.presentation.BinaryRepresentation.BINARY_REPRESENTATION;
+import static org.assertj.core.presentation.HexadecimalRepresentation.HEXA_REPRESENTATION;
+import static org.assertj.core.presentation.UnicodeRepresentation.UNICODE_REPRESENTATION;
+
+public class CompositeRepresentation_Test extends AbstractBaseRepresentationTest {
```

Comment:
add GIVEN WHEN THEN and use `then` as indicated in https://github.com/assertj/assertj-core/blob/main/CONTRIBUTING.md